kraken-build/src/kraken/common/findpython.py

```diff
@@ -148,6 +148,13 @@ def get_candidates(
             if re.match(r"\d+\.\d+\.\d+$", item.name) and item.is_dir():
                 yield {"path": str(item / "bin" / "python"), "exact_version": item.name}
 
+    # pyenv (Windows)
+    pyenv_versions = (Path(os.getenv('PYENV')) / "versions").expanduser()
```

Comment:
```suggestion
    pyenv_versions = (Path(os.getenv("PYENV")) / "versions").expanduser()
```

Black will complain about this